prowler/providers/aws/services/rds/rds_cluster_non_default_port/rds_cluster_non_default_port.py

```diff
@@ -0,0 +1,36 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.rds.rds_client import rds_client
+
+
+class rds_cluster_non_default_port(Check):
+    def execute(self):
+        findings = []
+        default_ports = {
+            3306: ["mysql"],
+            5432: ["postgres"],
+            8182: ["neptune"],
+        }
```

Comment:
Why only this ones?

Reply:
Those are the only engines that support clustered instances, so the difference with the instance check is that here we have fewer engine types, which is why I list these ports differently.

You can check it [here](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds/client/create_db_cluster.html):

**Engine (string) – [REQUIRED]**

The database engine to use for this DB cluster.

**Valid for Cluster Type**: Aurora DB clusters and Multi-AZ DB clusters

**Valid Values**:
- aurora-mysql
- aurora-postgresql
- mysql
- postgres
- neptune

If I'm wrong with this assumption, please ping me, and I'll make the necessary changes.

Reply:
Okay, yes they are the same as the instances so I'll add them, thanks for the help! 😄